### PR TITLE
add file change age to status view

### PR DESCRIPTION
**Pull request by lttlrck**
This add file change age to the status view.

The line slices seem a little fragile, maybe they should be turned into function calls.

Only cursory testing, it seems ok, but I only really tested with what I use on a daily basis.

### CODE REVIEW

**Comment by asfaltboy** [comment]
More details in ticket #1019

File: core/interfaces/status.py [hunk 1]
```diff
@@ -235,7 +256,7 @@ class GsStatusOpenFileCommand(TextCommand, GitCommand):
 
     def run(self, edit):
         lines = util.view.get_lines_from_regions(self.view, self.view.sel())
-        file_paths = (line.strip() for line in lines if line[:4] == "    ")
+        file_paths = (line[5:].strip() for line in lines if line[5:8] == "   ")
```
Comment (asfaltboy) [inline core/interfaces/status.py]: I feel we should take this opportunity to clarify what these slices mean and, possibly, replace with some sort of regex/syntax-scope matching instead?

Comment (lttlrck) [reply core/interfaces/status.py]: yes, especially as I got it wrong... it should be `line[7:].strip()`

**Comment by randy3k** [comment]
@lttlrck I have changed the merge base to `dev`. It is our practice to merge the new features into `dev` for evaluation first.

**Comment by lttlrck** [comment]
autostash breaks it because it resets the file timestamps on pull. Branching is another problem, I had that in the back of my mind but wasn't too concerned about that case, but autostash is a bit of a deal breaker for me. It'll need some research.

File: core/interfaces/status.py (hunk 1 above)
Comment (randy3k) [reply core/interfaces/status.py]: it seems to break too when a file was modified 100 weeks ago.

**Comment by randy3k** [comment]
@lttlrck could you also fix the [tests](https://github.com/divmain/GitSavvy/blob/master/syntax/test/syntax_test_status.txt) for the status dashboard?

**Comment by stoivo** [comment]
What's the status on this?

**Comment by lttlrck** [comment]
> What's the status on this?

well I'm using it and it's useful despite the timestamps being reset on branch changes and pull (with auto-stash), stashing etc.

However I would like to colorize the the date, so green (recent) -> red (old) to help pick out the recent/old changes. But that means diving into themes which I don't have time for at the moment.

**Comment by lttlrck** [comment]
> @lttlrck could you also fix the [tests](https://github.com/divmain/GitSavvy/blob/master/syntax/test/syntax_test_status.txt) for the status dashboard?

Not sure what I'm doing wrong, I thought that last commit would be ok, but it looks like it's tripping up on the deleted file test. I'll have to take a look later. Also, apologies I did I push force - making numerous commits to figure out the test failure problem is nasty.  I assume that's ok though? I can't run the tests locally AFAIK,

**Pull request closed without merging**